partner_invoice_target/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models,fields
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_invoice_target = fields.Boolean()
```

Comment:
```suggestion
    is_invoice_issuer = fields.Boolean("Qualified Invoice Issuer")
```

Assign '適格請求書発行事業者' as the translation.